### PR TITLE
RN 39707

### PR DESCRIPTION
**Pull request by EdithLansens**
@tobedeprez, could you please review this pull request to confirm the accurate inclusion of release note 39707 in the user guide? Thanks in advance!

### CODE REVIEW

File: user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ Please note the following regarding suggestion events:
 
 - Suggestion events generated to indicate a behavioral anomaly are automatically cleared 2 hours after their creation time or their last update time. From DataMiner 10.2.11/10.3.0 onwards, they are also cleared in case a new behavioral change is detected that ends the previous anomalous behavioral change. For example, when an alarm was created for an anomalous level increase at 1 PM, and a behavioral change point is detected at 2 PM when the level drops again, then the alarm created at 1 PM will be closed at 2 PM.
 
+- From DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 39707-->, suggestion events generated to indicate a behavioral anomaly for [virtual functions](xref:srm_definitions#virtual-function) are generated on the parent element. Prior to DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9, these suggestion events are generated on the child element.
+
+- Suggestion events generated to indicate a behavioral anomaly for [dynamic virtual elements](xref:Dynamic_virtual_elements) are generated on the child element.
```
Comment (tobedeprez) [inline user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md:24]: Can we make these two points into one point? Technically, virtual functions are a special kind of virtual elements, so I feel the first point doesn't really make much sense without the first one

File: user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Monitoring_of_trend_patterns.md [hunk 2]
```diff
@@ -29,3 +29,9 @@ The following limitations apply:
   - DataMiner checks all changes made to parameters for which patterns are being monitored in real time. If there are more than 6000 parameter changes per second, the following notice will be displayed in the Alarm Console:
 
     *High load on pattern matching functionality: reduced pattern match accuracy.*
+
+Please note the following regarding suggestion events:
+
+- From DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 39707-->, suggestion events triggered when a specific pattern is detected for [virtual functions](xref:srm_definitions#virtual-function) are generated on the parent element. Prior to DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9, these suggestion events are generated on the child element.
```
Comment (tobedeprez) [inline user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Monitoring_of_trend_patterns.md:35]: same

File: user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Proactive_cap_detection.md [hunk 3]
```diff
@@ -52,3 +52,9 @@ The value of the suggestion event mentions what kind of issue is expected, e.g.
 From DataMiner Cube 10.3.7/10.4.0 onwards, when a trend is projected to reach the minimum or maximum value within a parameter's range, the value of the suggestion event mentions "Predicted maximum value" or "Predicted minimum value", followed by the actual maximum or minimum value of the parameter. The corresponding unit of measurement, as specified in the protocol, will also be included. <!-- RN 36440 -->
 
 Prior to DataMiner Cube 10.3.7/10.4.0, the value of the suggestion event mentions "Predicted above range violation" or "Predicted below range violation".
+
+Please note the following regarding suggestion events:
+
```
Comment (tobedeprez) [inline user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Proactive_cap_detection.md:57]: Same

**Review by tobedeprez — CHANGES_REQUESTED** [review]

File: user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md [hunk 4]
```diff
@@ -19,6 +19,8 @@ Please note the following regarding suggestion events:
 
 - Suggestion events generated to indicate a behavioral anomaly are automatically cleared 2 hours after their creation time or their last update time. From DataMiner 10.2.11/10.3.0 onwards, they are also cleared in case a new behavioral change is detected that ends the previous anomalous behavioral change. For example, when an alarm was created for an anomalous level increase at 1 PM, and a behavioral change point is detected at 2 PM when the level drops again, then the alarm created at 1 PM will be closed at 2 PM.
 
+- Suggestion events generated to indicate a behavioral anomaly for [dynamic virtual elements](xref:Dynamic_virtual_elements) are generated on the child element. However, from DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 39707-->, suggestion events for [virtual functions](xref:srm_definitions#virtual-function) are generated on the parent element. Prior to DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9, these suggestion events are generated on the child element as well.
```
Comment (EdithLansens) [inline user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md:22]: @tobedeprez, I've combined the two points as suggested. Is this more accurate?

Comment (tobedeprez) [reply user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md:22]: Yes, perfect.

**Review by tobedeprez — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**